public/mapbox.html

```diff
@@ -0,0 +1,453 @@
+<!--
```

Comment:
It would be a lot better if you don't embed an iframe, and instead directly create something like `map.component.ts` with the appropriate html in it. Given time constraints, though, this works so is good.